tools/e2etesting/steps/deployplatform.yml

```diff
@@ -22,7 +22,7 @@ steps:
   condition: and(eq(variables['ContainerRegistryServer'], ''), eq(variables['ContainerRegistryUsername'], ''), eq(variables['ContainerRegistryPassword'], ''))
   inputs:
     azureSubscription: 'IOT-OPC-WALLS-SP'
-    KeyVaultName: 'developPipelineKV'
```

Comment:
I think that the condition for this task should also have a negation of what is checked by next secrets loading task, i.e `startsWith(variables['BranchName'], 'release')`.

Reply:
Good catch

Reply:
I'll send a follow-up PR for this as it doesn't obstruct the pipeline and move forward with this to fix the CI/CD break.